src/ProjectTemplates/Microsoft.Extensions.AI.Templates/src/ChatWithCustomData/ChatWithCustomData-CSharp.Web/Program.Aspire.cs

```diff
@@ -0,0 +1,81 @@
+using Microsoft.Extensions.AI;
+using Microsoft.Extensions.VectorData;
+using ChatWithCustomData_CSharp.Web.Components;
+using ChatWithCustomData_CSharp.Web.Services;
+using ChatWithCustomData_CSharp.Web.Services.Ingestion;
+#if (IsOllama)
+#else // IsAzureOpenAI || IsOpenAI || IsGHModels
+using OpenAI;
+#endif
+#if (UseAzureAISearch)
+using Microsoft.SemanticKernel.Connectors.AzureAISearch;
+#elif (UseQdrant)
+using Microsoft.SemanticKernel.Connectors.Qdrant;
+#endif
+
+var builder = WebApplication.CreateBuilder(args);
+builder.AddServiceDefaults();
+builder.Services.AddRazorComponents().AddInteractiveServerComponents();
+
+#if (IsOllama)
+builder.AddOllamaApiClient("chat")
+    .AddChatClient()
+    .UseFunctionInvocation()
+    .UseLogging();
```

Comment:
```suggestion
```
This happens automatically with the Ollama client. See [Fix Ollama OpenTelemetry pipeline (CommunityToolkit/Aspire#509)](https://github.com/CommunityToolkit/Aspire/pull/509)